i18n.php

```diff
@@ -921,7 +921,7 @@
     'but' => 'Nhưng',
     'examples' => 'Dữ liệu',
     'feature' => 'Tính năng',
-    'given' => 'Biết|Cho',
+    'given' => '*|Biết|Cho',
```

Comment:
See the discussion in https://github.com/Behat/Gherkin/issues/103

I will update the script this weekend to read the new i18n file.